app/javascript/application.js

```diff
@@ -0,0 +1,9 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+
+/**
+ * Load all javascript needed to run the AdminLTE theme and
+ * all the interactions.
+ */
+import 'admin-lte'
```

Comment:
@heyapricot i was able to make the sidenav work again by importing it here after pinning it.